### PR TITLE
Added Support.md

### PR DESCRIPTION
**Pull request by Saumya-Sunder**
Added Support.md

Signed-off-by: Saumya Sunder <saumya.sunder@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<!-- NOTE!!! This is copied into README.md.  If you update it here, update it there as well.) -->
+-   Converse with us in our CORTX-Open Source Slack channel [![Slack](https://img.shields.io/badge/chat-on%20Slack-blue")](https://cortx.link/slack_invite) to interact with community members and gets your questions answered.
+-   Check out our [Youtube Channel.](https://cortx.link/videos)
+-   Join us in [Discussions](https://github.com/Seagate/cortx-hare/discussions) to ask, answer, and discuss topics with your fellow CORTX contributors!
+    - You can subscribe to the Community mailing list by following GitHub Discussions as shown below:
+      1. Click **Watch** button at the top-left of the repository.
+      2. Click *Custom ->* from the Notifications list.
+      3. Select *Discussions* from the Custom dialog box and click **Apply**. 
+-   Ask and answer questions in our [Frequently Asked Questions](FAQs.md) page.
+-   Attend our [Monthly Meet an Architect](doc/meetings/README.rst) meetings to learn about CORTX architecture and participate in a Question and Answer Session.
```
Comment (knekrasov) [inline SUPPORT.md:12]: The link is not working (404 is shown)

File: SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+<!-- NOTE!!! This is copied into README.md.  If you update it here, update it there as well.) -->
+-   Converse with us in our CORTX-Open Source Slack channel [![Slack](https://img.shields.io/badge/chat-on%20Slack-blue")](https://cortx.link/slack_invite) to interact with community members and gets your questions answered.
+-   Check out our [Youtube Channel.](https://cortx.link/videos)
+-   Join us in [Discussions](https://github.com/Seagate/cortx-hare/discussions) to ask, answer, and discuss topics with your fellow CORTX contributors!
+    - You can subscribe to the Community mailing list by following GitHub Discussions as shown below:
+      1. Click **Watch** button at the top-left of the repository.
+      2. Click *Custom ->* from the Notifications list.
+      3. Select *Discussions* from the Custom dialog box and click **Apply**. 
+-   Ask and answer questions in our [Frequently Asked Questions](FAQs.md) page.
```
Comment (knekrasov) [inline SUPPORT.md:11]: The link is not working (404 is shown)

File: SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+-   Attend our [Monthly Meet an Architect](doc/meetings/README.rst) meetings to learn about CORTX architecture and participate in a Question and Answer Session.
+-   If you'd like to contact us directly, drop us a mail at cortx-questions@seagate.com.
+-   Subscribe to the our [developer newsletter](https://cortx.link/cortx-dev-newsletter) to and stay up to date on the latest CORTX developments, news, and events.
+-   Learn how to integrate CORTX with other technologies [here](doc/integrations/README.rst).
```
Comment (knekrasov) [inline SUPPORT.md:15]: The link is not working (404 is shown)

**Review by knekrasov — CHANGES_REQUESTED** [review]
I doubt that we have to add a document with broken links. Probably the text or links must be updated.

File: SUPPORT.md (hunk 1 above)
Comment (Saumya-Sunder) [reply SUPPORT.md]: Fixed the link

File: SUPPORT.md (hunk 2 above)
Comment (Saumya-Sunder) [reply SUPPORT.md]: Fixed the link

File: SUPPORT.md (hunk 3 above)
Comment (Saumya-Sunder) [reply SUPPORT.md]: Fixed the link

**Review by knekrasov — APPROVED** [review]

**Review by mssawant — APPROVED** [review]

**Pull request merged**